references/video_classification/train.py

```diff
@@ -360,6 +364,12 @@ def parse_args():
     parser.add_argument("--dist-url", default="env://", type=str, help="url used to set up distributed training")
 
     # Prototype models only
+    parser.add_argument(
+        "--prototype",
```

Comment:
nit: about the naming, what if we want to use prototype datasets for training in the future? shall we restrict the name a little bit so that it only implies that we are loading prototype models?
But I'm ok with leaving it as it is right now and revisiting it in the future. It's up to you :D

Reply:
Let's address it separately. One of the original ideas was to switch references to the prototype APIs of Transforms and Datasets to facilitate testing. There are no BC guarantees on the reference scripts (as long as all previous models are reproducible), so we can make changes as needed.